README.md

```diff
@@ -1,4 +1,4 @@
-![UCDList version](https://img.shields.io/badge/UCDList-EN--1.5-green.svg)
+![UCDList version](https://img.shields.io/badge/UCDList-WD--1.6-yellow.svg)
```

Comment:
I'm a CI skeptic and am not sure what this does, but if this is "WD" rather "PEN", there's something wrong with the underlying CI code, and that would need to be fixed.